h2o-r/scripts/h2o-r-test-setup.R

```diff
@@ -182,16 +182,27 @@ function() {
         strict_version_check <- TRUE
     } else if (IS.RUNIT) {
         # source h2o-r/h2o-package/R. overrides h2o package load
-        to_src <- c("aggregator.R", "classes.R", "connection.R","config.R", "constants.R", "logging.R", "communication.R",
-                    "kvstore.R", "frame.R", "targetencoder.R", "astfun.R","automl.R", "import.R", "parse.R", "export.R", "models.R", "edicts.R",
-                    "coxph.R", "coxphutils.R", "gbm.R", "glm.R", "gam.R", "anovaglm.R", "glrm.R", "kmeans.R", "deeplearning.R", "randomforest.R", "generic.R",
-                    "naivebayes.R", "pca.R", "svd.R", "locate.R", "grid.R", "word2vec.R", "w2vutils.R", "stackedensemble.R", "rulefit.R", "modelselection.R",
-                    "predict.R", "xgboost.R", "isolationforest.R", "psvm.R", "segment.R", "tf-idf.R", "explain.R", "permutation_varimp.R", "extendedisolationforest.R",
-                    "upliftrandomforest.R", "infogram.R", "isotonicregression.R", "admissibleml.R", "decisiontree.R", "adaboost.R")
-
-        src_path <- paste(h2oRDir,"h2o-package","R",sep=.Platform$file.sep)
-        invisible(lapply(to_src,function(x){source(paste(src_path, x, sep = .Platform$file.sep))}))
+        # to_src <- c("aggregator.R", "classes.R", "connection.R","config.R", "constants.R", "logging.R", "communication.R",
+        #             "kvstore.R", "frame.R", "targetencoder.R", "astfun.R","automl.R", "import.R", "parse.R", "export.R", "models.R", "edicts.R",
+        #             "coxph.R", "coxphutils.R", "gbm.R", "glm.R", "gam.R", "anovaglm.R", "glrm.R", "kmeans.R", "deeplearning.R", "randomforest.R", "generic.R",
+        #             "naivebayes.R", "pca.R", "svd.R", "locate.R", "grid.R", "word2vec.R", "w2vutils.R", "stackedensemble.R", "rulefit.R", "modelselection.R",
+        #             "predict.R", "xgboost.R", "isolationforest.R", "psvm.R", "segment.R", "tf-idf.R", "explain.R", "permutation_varimp.R", "extendedisolationforest.R",
+        #             "upliftrandomforest.R", "infogram.R", "isotonicregression.R", "admissibleml.R", "decisiontree.R", "adaboost.R")
 
+        # src_path <- paste(h2oRDir,"h2o-package","R",sep=.Platform$file.sep)
+        # invisible(lapply(to_src,function(x){source(paste(src_path, x, sep = .Platform$file.sep))}))
+        devtools::load_all(paste(h2oRDir,"h2o-package",sep=.Platform$file.sep), export_all=FALSE)
+        
+        additional_imports <- c(
```

Comment:
Lists methods from h2o that are not exported but are needed in the tests.

Methods such as `cut.H2OFrame` are available as `cut` for users but we use `cut.H2OFrame` in the test which I think is reasonable but not necessary.

One function I'm not sure about is `h2o.logIt`. Should we export it?

Reply:
@tomasfryda Do you think this `h2o.logIt` method? https://github.com/h2oai/h2o-3/blob/96293940921872354e54dd4108bcd8fac5c07494/h2o-r/h2o-package/R/logging.R#L47

It seems to do nothing. So I think it is not necessary to export it. But I am not sure.

Reply:
Good point. I should've checked the implementation... I will remove the function. As far as I know it's used only in our test setup to print out the seed so I'll change that to `cat`.